rdkit/ML/UnitTestAnalyzeComposite.py

```diff
@@ -52,7 +52,7 @@ def test1_Issue163(self):
 
     self.assertEqual(res[0][0],'BALABANJ')
     self.assertEqual(res[1][0],'BERTZCT')
-    self.assertEqual(res[-1][0],'FR_UNBRCH_ALKANE')
+    self.assertEqual(res[-1][0],'VSA_ESTATE9')
```

Comment:
What is `res[-1][0]` on your local copy? Res is sorted, so you should get this result.